src/definition/bazel_goto_definition_provider.ts

```diff
@@ -41,33 +74,13 @@ export class BazelGotoDefinitionProvider implements DefinitionProvider {
 
     const range = document.getWordRangeAtPosition(position, LABEL_REGEX);
     const targetText = document.getText(range);
-    const match = LABEL_REGEX.exec(targetText);
-
-    const targetName = match[1];
-    // don't try to process visibility targets.
-    if (targetName.startsWith("//visibility")) {
-      return null;
-    }
 
-    const queryResult = await new BazelQuery(
-      getDefaultBazelExecutablePath(),
-      Utils.dirname(document.uri).fsPath,
-    ).queryTargets(`kind(rule, "${targetName}") + kind(file, "${targetName}")`);
+    const location = await targetToUri(targetText, Utils.dirname(document.uri));
 
-    if (!queryResult.target.length) {
-      return null;
-    }
-    const result = queryResult.target[0];
-    let location;
-    if (result.type === blaze_query.Target.Discriminator.RULE) {
-      location = new QueryLocation(result.rule.location);
-    } else {
-      location = new QueryLocation(result.sourceFile.location);
-    }
     return [
       {
         originSelectionRange: range,
-        targetUri: Uri.file(location.path),
+        targetUri: Uri.file(location.path + `:$location.line:$location.column`),
```

Comment:
I think I might not have pushed a change to this which reflects the right way to do it
```suggestion
        targetUri: Uri.file(location.path).with({fragment: `${location.line}:${location.column}`})

```